packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol

```diff
@@ -240,10 +218,32 @@ contract L1StandardBridge is StandardBridge, Semver {
         address _to,
         uint256 _amount,
         bytes calldata _extraData
-    ) external payable onlyOtherBridge {
+    ) external payable {
         finalizeBridgeETH(_from, _to, _amount, _extraData);
     }
 
+    /**
+     * @custom:legacy
+     * @notice Finalizes a withdrawal of ERC20 tokens from L2.
+     *
+     * @param _l1Token   Address of the token on L1.
+     * @param _l2Token   Address of the corresponding token on L2.
+     * @param _from      Address of the withdrawer on L2.
+     * @param _to        Address of the recipient on L1.
+     * @param _amount    Amount of the ERC20 to withdraw.
+     * @param _extraData Optional data forwarded from L2.
+     */
+    function finalizeERC20Withdrawal(
+        address _l1Token,
+        address _l2Token,
+        address _from,
+        address _to,
+        uint256 _amount,
+        bytes calldata _extraData
+    ) external {
```

Comment:
Confirming there is a modifier here:
https://github.com/ethereum-optimism/optimism/blob/5101faaf61b3e9303a71d80dd59f4a145f6de760/packages/contracts-bedrock/contracts/universal/StandardBridge.sol#L332